solr_plugins/Schlagwortfolgensuche/src/de/uni_tuebingen/ub/ixTheo/keywordChainSearch/KeywordChainMetric.java

```diff
@@ -0,0 +1,77 @@
+package de.uni_tuebingen.ub.ixTheo.keywordChainSearch;
+
+
+import java.util.ArrayList;
+
+
+public class KeywordChainMetric {
+
+    static int calculateCommonPrefixLength(final String s1, final String s2) {
+	final int minLength = Math.min(s1.length(), s2.length());
+	int commonPrefixLength = 0;
+
+	for (int i = 0; i < minLength; ++i) {
+	    if (s1.charAt(i) != s2.charAt(i))
+		return commonPrefixLength;
```

Comment:
You count i and commonPrefixLength. They have the same value. So, why not returning i?